Mendelugram/Mendelugram/View/GridPhoto/PhotoCollectionViewCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  PhotoCollectionViewCell.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class PhotoCollectionViewCell: UICollectionViewCell {
+
+    static let reuseIdentifier = String(describing: PhotoCollectionViewCell.self)
+
+    struct Input {
+        let photo: UIImage
+    }
+
+    @IBOutlet weak var photoImageView: UIImageView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    func configure(input: Input) {
```

Comment:
I think that `func configure(with input: Input)` would be more in align with standard Swift naming.